test/distributed/fsdp/test_fsdp_use_orig_params.py

```diff
@@ -216,6 +216,7 @@ def _get_sharding_strategy_from_str(
             raise ValueError(f"Invalid string: {sharding_strategy_str}")
         return sharding_strategy
 
+    @unittest.skipIf(not SM70OrLater, "need sm_70")
```

Comment:
I guess this is from running locally on sm < 70?

Reply:
Right, the scheduled CI node's CUDA capability is 5.2.